src/unify/backing_vec.rs

```diff
@@ -10,29 +11,35 @@ use super::{VarValue, UnifyKey, UnifyValue};
 #[allow(type_alias_bounds)]
 type Key<S: UnificationStore> = <S as UnificationStore>::Key;
 
+pub trait Measurable {
```

Comment:
Can we add a doc comment to this trait? Maybe explain what this `len` represents? (I realize the old method didn't have a comment, sorry, I'm a hypocrite)

Reply:
I would've named this trait `Length`, really.

Or rely on [`ExactSizeIterator::len`](https://doc.rust-lang.org/std/iter/trait.ExactSizeIterator.html#method.len) somehow.

Reply:
Yeah, I considered that, but it didn't quite seem accurate: I wanted a word for something that *had* a length, rather than *was* a length. It's only supposed to be used internally though, so it's not too important.